### PR TITLE
Validate GORE

### PR DESCRIPTION
**Pull request by Speedpete81**
# Validate [GORE](https://solscan.io/token/7ubcoM29SVN4L6egHh5vGr5VThgm53XXtDNb3beqGCi)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/GOREONSOL/status/1795508176206340237
- Coingecko/ CMC URL (If available): https://coinmarketcap.com/dexscan/de/solana/BHoVB3TejfjR7R3LjUCc6Y4DJ8ZsaUNWgUjtfZBcg5ba/

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by Speedpete81** [comment]
Hi guys, i hope i did it correct. did not know if i needed to update the validated-tokens!
Thanks,
Pete

**Review by Speedpete81 — COMMENTED** [review]
dont know if correct!

File: pull_request_template.md
Comment (Zpa2024) [inline pull_request_template.md:1]: all good

Comment (Speedpete81) [reply pull_request_template.md:1]: ok

**Comment by Jymfletzfur** [comment]
PR inappropriately attempts to alter files other than the token csv

**Pull request closed without merging**